contents/handbook/growth/marketing/in-app.md

```diff
@@ -0,0 +1,41 @@
+---
+title: In-App
+sidebar: Handbook
+showTitle: true
+---
+
+Occasionally, we use in-app messages to tell users about certain things. We recognize that in-app messages can be intrusive and we want to avoid spamming our users with too many of them, too frequently. For that reason, we're judicious about the way in which we use them. 
+
+We currently don't have a separate system for tracking in-app messages, so Marketing currently owns the channel and is responsible for ensuring that messages aren't used excessively.
+
+## Types of in-app message
+
+Currently, there are three ways in which we can send in-app messages.
+
+- **Notification bar:** A message displayed across the top of the page, activated using the [Notification Bar](https://github.com/PostHog/notification-bar-app) app.
+- **In-app prompt:** A customizable pop-up which can be targeted to certain URLs and made to appear in the center of the page, or anchored in the corner. Activated using the `prompt` feature flag. [More info](/handbook/product/in-app-prompts).
```

Comment:
Ideally this would also be able to be targeted to a certain element a la the github notice Michael posted about in Slack. cc @lharries